packages/hyperflux/functions/StateFunctions.ts

```diff
@@ -142,12 +142,11 @@ export function syncStateWithLocalStorage<S, E extends Identifiable>(
           if (storedValue !== null && storedValue !== 'undefined') state[key].set(JSON.parse(storedValue))
```

Comment:
We are still calling `.set` here synchronously, which seems to cause onSet to be called without a rootState set